vNext/shared/AppInsightsCore/src/JavaScriptSDK/AppInsightsCore.ts

```diff
@@ -364,5 +388,6 @@ class ChannelController implements ITelemetryPlugin {
 }
 
 const validationError = "Extensions must provide callback to initialize";
+const validationErrorInExt = "Channels must be provided through config.channels only";
 const ChannelControllerPriority = 500;
-const duplicatePriority = "One or more extensions are set at same priority";
```

Comment:
Are you not checking for duplicated priorities anymore?

Reply:
I think the current behavior is for them to execute in order they are added if they have identical priorities. @jpiyali can correct if wrong

Reply:
yes, we dont check for duplicated priorities anymore. Abhi or you had raised that this was causing issues when I added the check. The string is unused.